bundle/language_en.properties

```diff
@@ -665,14 +665,23 @@ angal.hospital.visitendhour.txt
 angal.hospital.visitstarthour.txt                                                                      = Visit start hour
 angal.lab.allnegative.txt                                                                              = All negative.
 angal.inventory.allproduct.radio                                                                       = All products
+angal.inventory.anotherinventoryonthiswardstillanotheroneisinprogress                                  = Another inventory on this ward still another one is in progress
```

Comment:
First the suffix on this entry should be ".msg".   Second, the text of the message isn't very clear; actually it doesn't make any sense.